src/map.h

```diff
@@ -51,6 +51,8 @@ typedef std::vector< std::pair< item*, int > > itemslice;
 typedef std::string items_location;
 struct vehicle_prototype;
 using vproto_id = string_id<vehicle_prototype>;
+class vehicle_group;
+using vgroup_id = string_id<vehicle_group>;
```

Comment:
Please don't rename types like this, it just makes it harder to track down what they are.